src/test/resources/jsonComplaintExamples/review.json

```
@@ -0,0 +1,20 @@
+{
+  "creationDate": "1952-02-17",
+  "complaint": {
+    "complaintType": "REVIEW",
+    "reporterDetails": {
+      "applicantType": "APPLICANT",
+      "applicantName": "occaecat minim",
+      "applicantNationality": "Guatemala",
+      "applicantDob": "2020-10-03",
+      "applicantEmail": "abc@uevptde.com",
+      "applicantPhone": "01783 634807"
```

```suggestion
      "applicantPhone": "01632 960000"
```

Suggest updating this to a fake number approved by Ofcom. I'll also raise a ticket to fix up the others 

https://www.ofcom.org.uk/phones-telecoms-and-internet/information-for-industry/numbering/numbers-for-drama

---

Phone Number Changed